src/main/resources/knowledge_environment.graphqls

```diff
@@ -143,4 +144,5 @@ type ParticipantSummaryDataset {
   participantId: String
   redcapId: String
   tissueType: String
+  clinicalData: String

```

Comment:
This is a bit odd, because technically speaking this allows for each of the two query to pull the sum total of the summary and clinical data.. Perhaps in the future we may want to do so. This form does allow us to still have a clean separation of concerns on the front-end. I think this will be important as the amount of clinical data grows.